www/src/templates/template-starter-page.js

```diff
@@ -60,9 +60,10 @@ class StarterTemplate extends React.Component {
             <Helmet>
               <title>{`${repoName}: Gatsby Starter`}</title>
               <meta
-                name="og:image"
-                content={screenshotFile.childImageSharp.fluid.src}
+                property="og:image"
+                content={`../logo.svg`}
```

Comment:
Why this change?

Reply:
We have https://www.gatsbyjs.org/monogram.svg in https://github.com/gatsbyjs/gatsby/tree/master/www/static if that helps? Also could put a `PNG` or `JPG` version of the monogram or full logo (including the "Gatsby" wordmark) in there?

Reply:
…but I agree with Dustin, we should pass on the screenshot of the starter here?!

Reply:
reverted back to `screenshotFile.childImageSharp.fluid.src`